__tests__/app.test.js

```diff
@@ -29,18 +29,49 @@ describe("GET /api/topics", () => {
     })
 })
 describe("GET /api", () => {
-    it("Should return a status code: 200", () => {
-         return request(app)
-             .get("/api")
-             .expect(200)
-    })
     it("Should return a status code: 200 and a json of the endpoints", async () => {
         const filePath = path.join(__dirname, '../endpoints.json')
-        const output = JSON.parse(await fs.readFile(filePath, 'utf-8'))
+        const expectedJSON = JSON.parse(await fs.readFile(filePath, 'utf-8'))
 
         const result = await request(app).get('/api')
         expect(result.status).toBe(200)
-        expect(result.body).toEqual(output)
+        expect(result.body).toEqual(expectedJSON)
     })
+})
 
+describe("GET /api/article/:article_id", () => {
+    it("Should return a status code: 200 and an article object", () => {
+        return request(app)
+             .get("/api/articles/1")
+             .expect(200)
+             .then(({ body }) => {
+                const { article } = body
+                expect(article).toEqual({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
We can be more explicit here and say exactly what number is going to come back 👍